<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppMan Client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/images/ALogoClient.png</normaloff>:/images/images/ALogoClient.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBoxServerDetails">
      <property name="title">
       <string>Server Details</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditIPAddress">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblPort">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPort">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string>4450</string>
         </property>
         <property name="maxLength">
          <number>7</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonConnect">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/images/Connect.png</normaloff>:/images/images/Connect.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>250</width>
        <height>60</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonDisconnect">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/images/Disconnect.png</normaloff>:/images/images/Disconnect.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>250</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonConnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDisconnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>disconnectClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Connect()</slot>
  <slot>connectClick()</slot>
  <slot>disconnectClick()</slot>
 </slots>
</ui>
